recipes/apply_defaults/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "apply_defaults" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8c1bc511a0368dabe1af4d80b97186296e25182d7e371d920a9633cf6a2a385
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >3
+    - pip
+  run:
+    - python >3
+
+test:
+  imports:
+    - apply_defaults
+
+about:
+  home: https://github.com/bcb/apply_defaults
+  license: MIT
+  license_family: MIT
+  license_file: https://github.com/bcb/apply_defaults/blob/master/LICENSE
```

Comment:
Same here. Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually